doc/visual-programming/source/widgets/classify/randomforest.rst

```diff
@@ -1,9 +1,9 @@
-Random Forest
-=============
+Random Forest Classification
+============================
 
 .. figure:: icons/random-forest.png
 
-Random forest learner
+Random forest classification
 
```

Comment:
Gah! We're very inconsistent in this. Classification is not quite right, but so far we have 'classifier', 'learner' and 'learning algorithm here'. Anyone pro standardization?